smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/customtraits/TestEquatableConformanceTrait.kt

```diff
@@ -0,0 +1,13 @@
+package software.amazon.smithy.swift.codegen.customtraits
+
+import software.amazon.smithy.model.node.Node
+import software.amazon.smithy.model.shapes.ShapeId
+import software.amazon.smithy.model.traits.Trait
+
+class TestEquatableConformanceTrait : Trait {
```

Comment:
This trait is applied to shapes that need to have Equatable conformance added to them for protocol tests.